utils/authorization_test.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+type RoleState struct {
+	RoleName   string `json:"roleName"`
+	Permission string `json:"permission"`
+	ShouldHave bool   `json:"shouldHave"`
+}
+
+func mockConfig() *model.Config {
+	config := model.Config{}
+	config.SetDefaults()
+	return &config
+}
+
+func mapping() map[string]map[string][]RoleState {
+
+	policiesRolesMapping := make(map[string]map[string][]RoleState)
+
+	raw, err := ioutil.ReadFile("./policies-roles-mapping.json")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I'd recommend `require.NoError(t, err)` instead. More concise and has better feedback.

Reply:
@ccbrown Switched from `panic` to `require.NoError`.